snap/snapcraft.yaml

```diff
@@ -0,0 +1,489 @@
+    write:
+      - $HOME/.local/share/icons
+  desktop:
+    mount-host-font-cache: false
+  shmem:
+    interface: shared-memory
+    private: true
+
+hooks:
+  configure:
+    plugs:
+      - opengl
+
+parts:
+  launcher:
+    after: [heroic]
+    plugin: nil
+    source: $CRAFT_PROJECT_DIR/snap/local/src
+    override-build: |
+      mkdir -p $CRAFT_PART_INSTALL/usr/bin
+      cp * $CRAFT_PART_INSTALL/usr/bin/
+    stage-packages:
+      - python3-gi
+      - gir1.2-gtk-3.0
+    stage:
+      - bin/*
+      - usr/lib/python3/dist-packages/*
+      - usr/share/mime/*
+      - usr/share/icons/*
+      - usr/lib/**/gtk-3.0*
+      - etc/gtk-3.0*
+      - usr/share/*/gir1.2-gtk-3.0*
+      - usr/**/libfontconfig*
+      - usr/**/cairo-1*
+      - usr/**/libatspi*
+      - usr/**/libavahi-client*
+      - usr/**/libavahi-common*
+      - usr/**/libcairo-gobject*
+      - usr/**/libcairo*
+      - usr/**/libcolord*
+      - usr/**/libcups*
+      - usr/**/libdatrie*
+      - usr/**/libdconf*
+      - usr/**/libdeflate*
+      - usr/**/libepoxy*
+      - usr/**/libfribidi*
+      - usr/**/*girepository*
+      - usr/**/libgraphite2*
+      - usr/**/libgtk-3*
+      - usr/**/libharfbuzz*
+      - usr/**/libjbig*
+      - usr/**/libjpeg*
+      - usr/**/liblcms2*
+      - usr/**/libpango-1*
+      - usr/**/libpangocairo-1*
+      - usr/**/libpangoft2-1*
+      - usr/**/libpangoxft-1*
+      - usr/**/libpixman-1*
+      - usr/**/libthai*
+      - usr/**/libtiff*
+      - usr/**/libwayland-cursor*
+      - usr/**/libwebp*
+      - usr/**/libxcb-render*
+      - usr/**/libxkbcommon*
+    prime:
+      - -etc/dconf
+      - -etc/gtk*
+      - -etc/init*
+      - -etc/ld*
+      - -etc/vulkan
+      - -etc/X11
+      - -etc/xdg
+
+  # alsa-mixin:
+  #   plugin: dump
+  #   source: https://github.com/diddlesnaps/snapcraft-alsa.git
+  #   source-subdir: snapcraft-assets
+  #   source-depth: 1
+  #   build-packages:
+  #     - libasound2-dev
+  #   stage-packages:
+  #     - libasound2
+  #     - libasound2-plugins
+  #     - yad
+  #   stage:
+  #     # restrict to only audio-related files - you need to ensure
+  #     # that gtk3 is staged for yad to work correctly, to prompt
+  #     # users to connect the alsa plug or proceed with pulseaudio.
+  #     #
+  #     # This helps prevent symbol conflicts in situations where
+  #     # you're using a non-default library, such as those that the
+  #     # gnome-3-34 extension for core18 provides.
+  #     - etc/asound.conf
+  #     - snap/command-chain/alsa-launch
+  #     - usr/bin/yad*
+  #     - usr/lib/*/alsa-lib
+  #     - usr/lib/*/libasound*
+  #     - usr/lib/*/libasyncns*
+  #     - usr/lib/*/libdnsfile*
+  #     - usr/lib/*/libFLAC*
+  #     - usr/lib/*/libjack*
+  #     - usr/lib/*/libpulse*
+  #     - usr/lib/*/libsamplerate*
+  #     - usr/lib/*/libsndfile*
+  #     - usr/lib/*/libspeex*
+  #     - usr/lib/*/libvorbis*
+  #     - usr/lib/*/pulseaudio
+  #     - usr/share/alsa
```

Comment:
Probably want to remove entries like this completely if they're not needed. Otherwise you'll have people go "Why is this here again?" a year from now

Reply:
Okay... will clean it up